vision/getting-started/image_editing.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<p>Add a hyperlink to the documentation for these parameters.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Docs are a WIP, I will add the hyperlink once they're complete and publicly available</p>